pkg/management/component.go

```diff
@@ -585,17 +585,25 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 	}
 
 	// when the email changes, set the EmailVerified to false
-	if user.Email.Defined && (user.Email.Value == nil || c.isUpdated(user.Email.Value, oldUserKc.Email)) {
-		var verified = false
-		user.EmailVerified = &verified
+	if user.Email.Defined {
+		if user.Email.Value == nil {
+			var verified = false
+			user.EmailVerified = &verified
+		} else if c.isUpdated(user.Email.Value, oldUserKc.Email) {
+			if err = c.checkEmailNotUsed(ctx, realmName, userID, *user.Email.Value); err != nil {
+				c.logger.Info(ctx, "msg", "Can't update email with email of an already existing account", "email", *user.Email.Value)
```

Comment:
To avoid that support leak the info avout email already in use, we could allow to set it but send an email which state the email is already in use. It would use the same logic in account and management.